docs/src/algorithms/discretization.md

```diff
@@ -5,3 +5,55 @@ discretize
 DiscretizationMethod
 FIST
 ```
+# FIST  
+Examples for the FIST algorithm. 
+
+### Usage  
+  
+```@example overview
+# Import required Libraries
+using GeoStats
+using Plots; gr(size=(300,300))
+
+# Function to read Polygon from given values
+function readpoly(T, fname) 
+  open(fname, "r") do f
+    # read outer chain
+    n = parse(Int, readline(f))
+    outer = map(1:n) do _
+      coords = readline(f)
+      x, y = parse.(T, split(coords))
+      Point(x, y)
+    end
+
+    # read inner chains
+    inners = []
+    while !eof(f)
+      n = parse(Int, readline(f))
+      inner = map(1:n) do _
+        coords = readline(f)
+        x, y = parse.(T, split(coords))
+        Point(x, y)
+      end
+      push!(inners, inner)
+    end
+
+    # return polygonal area
+    PolyArea(outer, inners)
+  end
+end
+```  
+Now, we will read point values from an example file `smooth1.line`, and construct a polygon from those files using the `readpoly` function in the following manner:  
```

Comment:
```suggestion
Now, we will read point values from an example file `smooth1.line`, and construct a polygon from those points using the `readpoly` function:  
```
I think you meant "points". I would remove the "in the following manner" to make the sentence lighter.